server/export.go

```diff
@@ -109,7 +109,7 @@ func ExportCmd(appExporter types.AppExporter, defaultNodeHome string) *cobra.Com
 			return nil
 		},
 	}
-
+	cmd.SetOutput(os.Stdout)
```

Comment:
This should call cmd.OutOrStdout()

Plus, please ensure that errors go to stderr (i.e. cmd.SetError(cmd.ErrOrStdErr()))